<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spygee</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="logTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>911</width>
      <height>101</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
   </widget>
   <widget class="QTreeView" name="apiTreeView">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>40</y>
      <width>911</width>
      <height>391</height>
     </rect>
    </property>
    <property name="rootIsDecorated">
     <bool>true</bool>
    </property>
    <property name="animated">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>911</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Apigee Management API URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="urlLineEdit">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Reset settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="goButton">
       <property name="text">
        <string>GO!</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
